vast/CMakeLists.txt

```diff
@@ -1,25 +1,36 @@
-option(VAST_ENABLE_VAST "Build the standalone vast binary"
-       "${VAST_IS_NOT_SUBPROJECT}")
-add_feature_info("VAST_ENABLE_VAST" VAST_ENABLE_VAST
-                 "build the standalone vast binary.")
-
-if (NOT VAST_ENABLE_VAST)
-  return()
-endif ()
-
-add_executable(vast vast.cpp)
-VASTTargetEnableTooling(vast)
-target_link_libraries(vast PRIVATE vast::internal vast::libvast)
-VASTTargetLinkWholeArchive(vast PRIVATE vast::libvast_builtins)
+add_executable(tenzir tenzir.cpp)
+VASTTargetEnableTooling(tenzir)
+target_link_libraries(tenzir PRIVATE vast::internal vast::libvast)
+VASTTargetLinkWholeArchive(tenzir PRIVATE vast::libvast_builtins)
 install(
-  TARGETS vast
+  TARGETS tenzir
   DESTINATION "${CMAKE_INSTALL_BINDIR}"
   COMPONENT Runtime)
-add_executable(vast::vast ALIAS vast)
+add_executable(vast::tenzir ALIAS tenzir)
+
+macro (create_tenzir_symlink name)
+  add_custom_command(
+    OUTPUT "${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}/${name}"
+    COMMAND ${CMAKE_COMMAND} -E create_symlink "./tenzir"
+            "${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}/${name}"
+    DEPENDS tenzir
+    COMMENT "Creating ${name} symlink")
+  add_custom_target(
+    "${name}" ALL DEPENDS "${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}/${name}")
+  install(
+    CODE "
+    execute_process(COMMAND
+      \"${CMAKE_COMMAND}\" -E create_symlink \"./tenzir\"
+      \"\${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/${name}\")")
```

Comment:
Before merging, we must manually check that CPack handles this correctly by inspecting the CI-generated packages.